.github/workflows/test.yaml

```diff
@@ -0,0 +1,22 @@
+name: Tests
+
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out PR branch
+        uses: actions/checkout@v2
+
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          distribution: temurin
+          java-version: 11
+
+      - name: Run Android tests
+        run: ./gradlew :android:test --console=rich
```

Comment:
This won't run any tests since the android tests are only run with `./gradlew connectedAndroidTest`, and those test require an emulator. But we can fix that in a follow-up PR once we figure out the steps.